toolkit/docs/how_it_works/3_package_building.md

```diff
@@ -124,6 +126,8 @@ Edges in the graph represent dependencies. A package cannot be installed (`run`)
 
 Once all packages have been added to the graph, the inter-package dependencies are added. For each `BuildRequires` in a package an edge is created from the current `build` node to the `run` node associated with the required package. The same is done for each `Requires`, but from the current `run` node instead of the `build` node.
 
+Edges will not be created for `Requires` between two `TypeRun` nodes that have the same `RpmPath` since the dependency will automatically be met when the rpm is installed. If the edges were created it could introduces extra cycles that would need to be solved.
```

Comment:
>Edges will not be created for `Requires` between two `TypeRun` nodes that have the same `RpmPath` [](start = 0, length = 97)

Is that just another way of saying that nodes for subpackages from one spec won't be connected?

Reply:
This is a very specific case - a single RPM in our graph can be represented by multiple run nodes (e.g. if a single rpm provides multiple things). With this PR we no longer create edges between run nodes that are representing the same rpm.